gradle/module.go

```diff
@@ -0,0 +1,25 @@
+package gradle
+
+import "fmt"
+
+// Module ...
+type Module struct {
+	project Project
+	name    string
+	tasks   []Task
+}
+
+// SetTask ...
+func (module Module) SetTask(name string) *Task {
```

Comment:
SetTask -> NewTask / CreateTask ?

Reply:
SetTask is because I don't want to create a new task, but want to set a task for the module that will be ran